book/proposal/proposal.md

```diff
@@ -0,0 +1,99 @@
+# Proposal for Design Think for Developers
+### By: Cory Gwin
+
+## Overview
+Technology is simply a tool for solving a problem. Arguably the most valuable tool any developer has in their tool chest is the ability to understand, communicate about, break down a problem and provide a proven solution. Most organization find this incredibly difficult to do and often organization focus on how fast they can ship instead of how well their solutions are received. Oddly, as terrible as we are at this part of our jobs, our problem-solving tools get nowhere near the attention as our text editor skills, the scripting language of choice or deployment skills and other tools in our development war chest. Instead, we hand it off to other parts of the organization and later complain about them throwing requirements over the wall without asking if we should break down the wall.
+
+It is time to break down these artificial boundaries and take responsibility for the solutions we build. The agile manifesto states we should prefer customer collaboration over contract negotiation. As developers we often leave the customer collaboration to others in our organization, leaving only short user stories or issues to gather context. This leads to a number of problems including:
+
+- Downstream communication issues.
+- Misunderstood requirements.
+- Incorrect implementations.
+- Developer frustration.
+- Scope change or creep. 
+- Lost context over time.
+- Code that does not properly convey intent.
+- Unintended consequences of poorly modeled data.
+- Incorrect prioritization.
+
+Our workflows and structures create communication barriers we must break down in order to move our implementations closer to their domains. As developers, our goals should include moving ourselves and our code closer to the domain. This requires understanding not only the problem being solved by a feature but also, more broadly an understanding of the field we are working in.
```

Comment:
"As developers, our goals should include moving ourselves and our code closer to the domain." :heart: